microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -247,23 +247,25 @@ The family group organizer may stop the data collection from the family safety p
 <!-- ====================================================================== -->
 ## Geolocation
 
-Microsoft Edge supports the [Geolocation API](https://w3.org/TR/geolocation-api), which allows websites to access your location information with your permission.  Websites may ask for your location, for example, when trying to find the closest coffee shop near you.  Microsoft Edge always asks for your permission before granting websites access to your location.  To manage the permission or to always block sites from accessing your location, navigate to `edge://settings/content/location`.
+While you browse the web, websites may request your device’s location from Microsoft Edge. Data about your device’s location can be either precise or imprecise. For example, precise location is requested to provide driving direction to/from your specific location. Imprecise location may be requested to provide search results, news, and weather relevant to your general location.
```

Comment:
change curly quotes to smart quotes, at least in this h2 section
```suggestion
While you browse the web, websites may request your device's location from Microsoft Edge. Data about your device's location can be either precise or imprecise. For example, a precise location is requested to provide driving directions to or from your specific location. An imprecise location may be requested to provide search results, news, and weather relevant to your general location.
```

Reply:
Changed in a single Commit